src/java.xml/share/classes/module-info.java

```diff
@@ -229,16 +249,105 @@
  * </li>
  * </ul>
  *
+ * <h2 id="PP">Property Precedence</h2>
+ * Properties in this module can be set in multiple ways, including using
+ * the API properties, system properties, <a href="#ConfigurationFile">JAXP Configuration File</a>,
+ * the {@link javax.xml.XMLConstants#FEATURE_SECURE_PROCESSING FEATURE_SECURE_PROCESSING}
+ * (hereafter referred to FSP), and the default values. The order of precedence
+ * for the configuration sources is defined as follows, with earlier ones overriding the later:
```

Comment:
"using the API properties" is bit confusing. I think you mean that properties can be set via the APIs. 

".. and the default values". I think it would be clearer to say that properties have default values when not set by previous four ways.

Reply:
Updated accordingly in additional commits.